lib/inquirer-prompt.js

```diff
@@ -0,0 +1,43 @@
+const inquirer = require('inquirer');
+const chalk = require('chalk');
+
+module.exports = function(questions, config) {
+	inquirer.prompt(questions).ui.process
+  .reduce(function (newOpts, ans) {
```

Comment:
Something is wrong here with the indentation. I guess we just don't have an eslint rule to enforce it.